scripts/schema-publish.sh

```
@@ -0,0 +1,78 @@
+    newestCommitDate=$(git log -1 --format="%ad" --date=short "$schemaDir/$schema")
+
+    # the newest date across a schema and all its dependencies is its date stamp
+    if [ "$newestCommitDate" \> "$maxDate" ]; then
+      maxDate=$newestCommitDate
+    fi
+
+    base=$(basename $schema '.yaml')
+    # Add the replacement for this schema's placeholder to list of sed commands.
+    sedCmds+=("s/${base}\/WORK-IN-PROGRESS/${base}\/${maxDate}/g")
+
+    publish_schema "$schema" "$maxDate" $(printf '%s;' "${sedCmds[@]}")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)